bestiary/core/schema.py

```diff
@@ -108,6 +113,17 @@ class Meta:
         description = "Transaction objects representing atomic sets of operations modifying the database."
 
 
+class EcosystemType(DjangoObjectType):
```

Comment:
Can we add documentation to all these classes so the documentation is available in the schema?